components/sup/src/main.rs

```diff
@@ -801,17 +801,18 @@ RCFaO84j41GmrzWddxMdsXpGdn3iuIy7Mw3xYrjPLsE="#,
                                                  application_environment: None,
                                                  binds:                   None,
                                                  specified_binds:         None,
-                                                 binding_mode:            None,
+                                                 binding_mode:            Some(1),
```

Comment:
Are we really required to refer to numbers here in the protobuf types?

Reply:
No, I just needed to use `into` on the enum. Fixed in 26e8c1bcf29de69db2a9576043da8c18eb29b9d1 along with your other suggestions.